### DISCUSSION TITLE
Bun support as package manager, pre-release channel?

### DISCUSSION THREAD

**Discussion by Ragura** [nuxtrdev/nuxtr-vscode#46] [Questions]
### Environment

Using Bun 1.0.1

### What went wrong?

First of all, thanks for providing this excellent extension, it's made my life working with Nuxt a lot easier!

I noticed you've worked on adding support for Bun as the package manager. Is there a pre-release channel for the extension where this support is available? If not, I'll just wait for a proper release :)

### How to reproduce it?

Try selecting Bun as package manager.

**Comment by adhamfarrag** [comment]
Hey @Ragura, thanks for your kind words! :)

Exactly as you described, Bun support is done but we're waiting for proper release with all planned features combined together. This should happen next week or max the week after. 🤞

We haven't added any pre-release to the next release, but happy to guide you towards building and running a local version of the extension if you really need it. 

```bash
# install vsce package globally
npm install -g @vscode/vsce

# clone Nuxtr and install pnpm if you haven't already
git clone https://github.com/nuxtrdev/nuxtr-vscode/
npm install -g pnpm

# uninstall the extension from VSCode.

# package the extension and remove
pnpm package 

# Install the local version from the vsix file
code --install-extension nuxtr-vscode-0.1.11.vsix
```

To make sure the steps are clear regardless your situation, if `code` doesn't work, it means you need to use `Shell Command: Install 'code' command in PATH` from Command Palette to install it.

**Reply by Ragura** [reply]
Thanks for the quick reply and abundant information! I'll hold off on installing until a proper release, looking forward to it! :)

**Comment by adhamfarrag** [comment]
Hey @Ragura, sorry for the long wait.. Nuxtr 0.2.0 has been shipped with Bun Support. :)

**Reply by Ragura** [reply]
Excellent! Thanks for the continued work! 🙏
